typescript/sdk/src/core/EvmCoreModule.ts

```diff
@@ -90,12 +93,96 @@ export class EvmCoreModule extends HyperlaneModule<
     const transactions: AnnotatedEV5Transaction[] = [];
 
     transactions.push(
+      ...(await this.createUpdateDefaultIsmTx(actualConfig, expectedConfig)),
```

Comment:
not sure if i love this name, but lmk if clear

Reply:
it follows the current convention so i think it's fine

Reply:
but please make plural and consider name order update

Reply:
Order matters here be because if you transfer ownership first. You wouldn’t be able to do other actions

Reply:
I like the descriptive name. My general feeling is keep names as short as they can reasonable be in small scopes (like a local var in a fn) but descriptive in larger scopes (like a fn exported from a file, meaning it's available to the entire project / monorepo)

Reply:
> Order matters here be because if you transfer ownership first. You wouldn’t be able to do other actions

i just meant `createDefaultIsmUpdateTx` instead of `createUpdateDefaultIsmTx`

Reply:
> I like the descriptive name. My general feeling is keep names as short as they can reasonable be in small scopes (like a local var in a fn) but descriptive in larger scopes (like a fn exported from a file, meaning it's available to the entire project / monorepo)

if we strive to keep names as short as they can, won't this eventually lead to readability/maintainability issues? IMO our var name choices should simply be a product of their types, e.g. `const submitterBuilder: SubmitterBuilder` and **not** `const submitter: SubmitterBuilder` etc @jmrossy

Reply:
@nbayindirli I see your point and seems you like feel a bit more strongly about it than me. Names that match types sgtm :)